backend.go

```diff
@@ -129,27 +131,42 @@ func (b *jwtAuthBackend) getProvider(config *jwtConfig) (*oidc.Provider, error)
 	return provider, nil
 }
 
-// getKeySet returns a new JWKS KeySet based on the provided config.
-func (b *jwtAuthBackend) getKeySet(config *jwtConfig) (oidc.KeySet, error) {
+// getJWTValidator returns a new JWT validator based on the provided config.
+func (b *jwtAuthBackend) getJWTValidator(config *jwtConfig) (*jwt.Validator, error) {
```

Comment:
Nit: Maybe we should take this chance to rename the func to `jwtValidator()`. Getters shouldn't need the `get*` prefix on their func names.

Reply:
Yep, might as well do that now. Done in 722a769.